src/PrestaShopBundle/Controller/Admin/ProductController.php

```diff
@@ -455,9 +456,18 @@ public function formAction($id, Request $request)
         }
 
         $productAdapter = $this->get('prestashop.adapter.data_provider.product');
-        $product = $productAdapter->getProduct($id);
+        try {
+            $product = $productAdapter->getProduct($id);
+        } catch (LogicException $e) {
+            $product = null;
+        }
 
         if (!$product || empty($product->id)) {
+            $this->addFlash(
+                'warning',
+                $this->trans('The product you are trying to access doesn\'t exist', 'AdminLoginNotification')
```

Comment:
```suggestion
                $this->trans('The product you are trying to access doesn\'t exist.', 'Admin.Orderscustomers.Notification')
```

Reply:
```suggestion
                $this->trans('The product you are trying to access doesn\'t exist', 'Admin.Catalog.Notification')
```
Hi @l-delin if the namespace is wrong then this one seems more adapted no? `Orderscustomers` has no relationship with the product page in theory

Reply:
Hello @jolelievre Yes, you're right - my bad. 😄  I'll edit my suggestion